include/openssl/trace.h

```diff
@@ -56,7 +56,8 @@ extern "C" {
 # define OSSL_TRACE_CATEGORY_STORE              14
 # define OSSL_TRACE_CATEGORY_DECODER            15
 # define OSSL_TRACE_CATEGORY_ENCODER            16
-# define OSSL_TRACE_CATEGORY_NUM                17
+# define OSSL_TRACE_CATEGORY_REF_COUNT          18
```

Comment:
Should it be 17?

Reply:
Please look again: 17 is now unused value and 18 is both OSSL_TRACE_CATEGORY_REF_COUNT and OSSL_TRACE_CATEGORY_NUM

Reply:
@t8m, you and @levitte decide since I am getting conflicting direction.

Reply:
No, we are not! Please look again at your change and at what @levitte suggested.

Reply:
Looking at https://github.com/openssl/openssl/pull/15353#discussion_r636634108 he suggested making it 18